cosmovisor/cmd/cosmovisor/cmd/version.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Cosmovisor version value. Set during build
+var Version string
+
+func isVersionCommand(args []string) bool {
+	return len(args) == 1 && strings.EqualFold(args[0], "version")
+}
+
+func printVersion() {
+
```

Comment:
you can remove maybe the new lines between the println.

Reply:
👍🏻